### DISCUSSION TITLE
FlowJo Group Assignment

### DISCUSSION THREAD

**Discussion by ace040686** [whitews/FlowKit#152] [Q&A]
Been looking at the documentation and was wondering if there was a way to open a FlowJo workspace session, and then have event data be grouped and labeled in that same workspace. My goal would be to run a cluster analysis of the data, and then group the results into a FlowJo workspace file for users to further examine/analyze.

**Comment by whitews** [comment]
Hi ace,

Not sure how this would be done. Currently the export to a FlowJo workspace is available only from a Session where there is only one group.  I've seen approaches before where new FCS files were generated with columns for a cluster number, but that isn't ideal. If you are aware of any FlowJo documentation for how sample data can be grouped by cluster results I'd be interested in having a look.

Thanks,
Scott

**Reply by ace040686** [reply]
Thanks, I am pretty new to the flow cyto and flowjo world so I am not sure of the inner workings. Not sure if I can provide insight on FlowJo related documentation. My goal is to retrieve analysis from open source libraries as shown in your flowkit, and put it into a FlowJo oriented file so that our scientists can look, verify, and even further explore the auto-generated findings.
